src/NexusMods.Collections/Verbs.cs

```diff
@@ -107,4 +119,91 @@ private static async Task<int> GatherCollectionDefinitions([Injected] IRenderer
 
         return 0;
     }
+
+    [Verb("create-collection", "Creates a collection")]
+    private static async Task<int> CreateCollection(
+        [Option("l", "loadout", "Loadout")] Loadout.ReadOnly loadout,
+        [Option("s", "slug", "Slug")] string slug,
+        [Injected] IConnection connection,
+        [Injected] IGameDomainToGameIdMappingCache mappingCache,
+        [Injected] ILoginManager loginManager,
+        [Injected] ILibraryService libraryService,
+        [Injected] TemporaryFileManager temporaryFileManager,
+        [Injected] JsonSerializerOptions jsonSerializerOptions,
+        [Injected] CancellationToken cancellationToken)
+    {
+        var entityIds = loadout.Db.Datoms(
+            (CollectionGroup.IsReadOnly, false),
+            (LoadoutItem.LoadoutId, loadout)
+        );
+
+        if (!entityIds.TryGetFirst(out var collectionGroupId)) return -1;
+
+        var collectionGroup = CollectionGroup.Load(loadout.Db, collectionGroupId);
+        if (!collectionGroup.IsValid()) return -1;
+
+        var userInfo = loginManager.UserInfo;
+        if (userInfo is null) return -1;
+
+        var users = User.FindByNexusId(connection.Db, userInfo.UserId.Value);
+        if (!users.TryGetFirst(out var user))
+        {
+            using var tx1 = connection.BeginTransaction();
+
+            var newUser = new User.New(tx1)
+            {
+                Name = userInfo.Name,
+                AvatarUri = userInfo.AvatarUrl ?? new Uri("https://example.org"),
+                NexusId = userInfo.UserId.Value,
+            };
+
+            var result = await tx1.Commit();
+            user = result.Remap(newUser);
+        }
+
+        using var tx = connection.BeginTransaction();
+
+        var (collectionManifest, collectionMetadata, collectionRevisionManifest) = CollectionCreator.CreateCollection(
+            tx: tx,
+            group: collectionGroup.AsLoadoutItemGroup(),
+            mappingCache: mappingCache,
+            author: user,
+            collectionSlug: CollectionSlug.From(slug)
+        );
+
+        await AddManifest(libraryService, temporaryFileManager, jsonSerializerOptions, tx, collectionManifest, collectionMetadata, collectionRevisionManifest, cancellationToken);
+
+        await tx.Commit();
+        return 0;
+    }
+
+    private static async ValueTask AddManifest(
```

Comment:
Add manifest to what? To Nexus? I would suggest using a more descriptive name for the function or adding a comment to define its scope